maintainers/maintainer-list.nix

```diff
@@ -4782,6 +4782,11 @@
     githubId = 19479662;
     name = "Kajetan Champlewski";
   };
+  millerjason = {
+    email = "mailings-github@millerjason.com";
+    github = "millerjason";
+    name = "Jason Miller";
```

Comment:
you also need to include your githubId

Reply:
Thanks - githubid added, rebased to avoid conflicts again.